src/js/main.js

```diff
@@ -0,0 +1,36 @@
+import auth from './auth';
+
+const authenticationPage = document.querySelector('#authentication-page');
+const signInButton = document.querySelector('#sign-in-button');
+const signOutButton = document.querySelector('#sign-out-button');
+const formButton = document.querySelector('#form-button');
+
+const authHandler = auth();
+
+const signIn = () => {
+  authenticationPage.classList.replace('flex', 'hidden');
+};
+
+const signOut = () => {
+  authenticationPage.classList.replace('hidden', 'flex');
+};
+
+const submitEvent = (event) => {
+  event.preventDefault();
+
+  signInButton.addEventListener('click', signIn, () => {
```

Comment:
This code is not needed, you are already handling the submit of the user, no need to add an evenlistener inside another eventlistner doing the same thing.

Reply:
Done